.github/workflows/release.yaml

```diff
@@ -35,5 +35,10 @@ jobs:
         run: |
           CHART_VERSION=$(yq '.version' Chart.yaml)
           EBS_CSI_DRIVER_VERSION=$(yq '.dependencies[] | select(.name == "aws-ebs-csi-driver") | .version' Chart.yaml)
+          echo "📦 Packaging base chart..."
+          helm package --version "${CHART_VERSION}" .
+          helm s3 push "datafy-agent-${CHART_VERSION}.tgz" datafyio-s3 --relative
+
+          echo "📦 Packaging with EBS CSI dependency version..."
           helm package --version "${CHART_VERSION}+ebscsi.${EBS_CSI_DRIVER_VERSION}" .
-          helm s3 push datafy-agent-*.tgz datafyio-s3 --relative
+          helm s3 push "datafy-agent-${CHART_VERSION}+ebscsi.${EBS_CSI_DRIVER_VERSION}.tgz" datafyio-s3 --relative
```

Comment:
i dont think it will work.
`datafy-agent-${CHART_VERSION}.tgz` and `datafy-agent-${CHART_VERSION}+ebscsi.${EBS_CSI_DRIVER_VERSION}.tgz` technically are the same sem version. because in sem version, after the '+' is metadata.
why you need to pack without the dependency? if you dont want to install the dep, there is a flag